### PR TITLE
[Docker Compose] Add new file for DB and Redis-only setup

### PR DESCRIPTION
**Pull request by YodaLightsabr**
## Summary of the problem

image

This file has been sitting in my sidebar for months, and it would be nice to check it in to git!

## Describe your changes
Add the `docker-compose.dbredisonly.yml` file for running HCB outside of Docker but running postgres and redis inside of Docker. I'm open to naming suggestions.

### CODE REVIEW

**Review by sampoder — DISMISSED** [review]
Docker-less, sigh.

**Review by garyhtou — DISMISSED** [review]

**Comment by garyhtou** [comment]
if you want to use this, i think you'll need to pass in a `-f` flag
`docker compose up -f docker-compose.dbonly.yml`

File: docker-compose.dbonly.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: '3'
```
Comment (garyhtou) [inline docker-compose.dbonly.yml]:
```suggestion
# To use this DB-only ("dockerless") setup, pass a `-f docker-compose.dbonly.yml` to docker compose.
# e.g. `docker compose up -f docker-compose.dbonly.yml`
version: '3'
```

**Review by garyhtou — APPROVED** [review]

**Comment by garyhtou** [comment]
🐳

**Pull request merged**